### ISSUE TITLE
Nodes outputting Processes don't work

### ISSUE THREAD

**Issue by neworderofjamie** [project-rig/nengo_spinnaker#4]
As well as functions and constant values, nodes can output Processes (nengo.processes.Process derived things). First problem occurs in builder https://github.com/project-rig/nengo_spinnaker/blob/master/nengo_spinnaker/builder/ensemble.py#L35 which can be simply fixed with a `not isinstance(connection.pre_obj.output, Process)` test (is that the best way?) but it then falls over in https://github.com/project-rig/nengo_spinnaker/blob/master/nengo_spinnaker/operators/value_source.py#L73 where I'm a little sketchy as to what's going on. The following example reproduces:

``` python
import nengo
import nengo_spinnaker
import numpy as np
import matplotlib.pyplot as plt

from nengo.processes import WhiteSignal

model = nengo.Network()
with model:
    inp = nengo.Node(WhiteSignal(60, high=5), size_out=2)
    pre = nengo.Ensemble(60, dimensions=2)
    nengo.Connection(inp, pre)

    pre_p = nengo.Probe(pre, synapse=0.01)

nengo_spinnaker.add_spinnaker_params(model.config)
model.config[inp].function_of_time = True

sim = nengo_spinnaker.SpiNNakerSimulator(model)
sim.run(10.0)
```

**Comment by neworderofjamie** [comment]
Actually, further consideration - have attempted fix in https://github.com/project-rig/nengo_spinnaker/tree/processes example seems to now crash the board...

**Comment by mundya** [comment]
Fixed by #5, thank you @neworderofjamie.

**Comment by pabogdan** [comment]
`WhiteNoise` init has changed and it's not working again, as you can see by the two plots attached. The value output by it tends to be stable for about  5-8 seconds, then outputs a different value.

**Note** changing scale to False has no effect on output.

``` python
class WhiteNoise(Process):
     def __init__(self, dist=None, scale=True):
```

``` python
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline

import nengo
```

``` python
from nengo.utils.functions import piecewise

__author__ = 'bogdanp'

import numpy as np
import nengo
from nengo.processes import WhiteNoise
import nengo_spinnaker

model = nengo.Network()
with model:
    # Input
    inp = nengo.Node(WhiteNoise(scale=True))
    pre = nengo.Ensemble(60, dimensions=1)
    nengo.Connection(inp, pre)

    pre_p = nengo.Probe(pre, synapse = 0.1)
```

``` python
nengo_spinnaker.add_spinnaker_params(model.config)
model.config[inp].function_of_time = True

sim = nengo_spinnaker.Simulator(model)

with sim:
    sim.run(5)
```

``` python
plt.figure(figsize=(12, 8))
plt.subplot(2, 1, 1)
plt.plot(sim.trange(), sim.data[pre_p].T[0], c='b', label='Pre')
plt.ylabel("Dimension 1")
plt.legend(loc='best')
```

```
<matplotlib.legend.Legend at 0x7f26d5a31d50>
```

![output_3_1](https://cloud.githubusercontent.com/assets/1470639/10611186/d0499b2c-7742-11e5-8d37-20fc057694b1.png)

``` python
sim_host = nengo.Simulator(model)
sim_host.run(5)

```

```
Simulation finished in 0:00:01.                                                 
```

``` python
plt.figure(figsize=(12, 8))
plt.subplot(2, 1, 1)
plt.plot(sim_host.trange(), sim_host.data[pre_p].T[0], c='b', label='Pre')
plt.ylabel("Dimension 1")
plt.legend(loc='best')
```

```
<matplotlib.legend.Legend at 0x7f26d5104690>
```

![output_5_1](https://cloud.githubusercontent.com/assets/1470639/10611175/c75ae462-7742-11e5-8ff6-048eb5a62d9f.png)

``` python

```
